### ISSUE TITLE
CModule: include file 'gum/gummetalarray.h' not found

### ISSUE THREAD

**Issue by andreafioraldi** [frida/frida-gum#389]
Hi,
I'm trying to write the transform of a Stalker in C.
On x86 I got this error:
```
frida.core.RPCException: Error: Compilation failed: In file included from module.c:4:
In file included from /frida/gum/gumstalker.h:5:
In file included from /frida/gum/arch-x86/gumx86writer.h:5:
/frida/gum/arch-x86/gumx86writer.h:5: error: include file 'gum/gummetalarray.h' not found
```
This sounds very strange cause this header exists in frida-gum and should be there. I'm using now the pip version (12.8.0).
Maybe it is a simple packaging issue.

**Comment by andreafioraldi** [comment]
To reproduce (on Linux x64) simply open the shell.
```
[Local::PID::28717]-> new CModule("#include <gum/gumstalker.h>")                                
Error: Compilation failed: In file included from module.c:1:
In file included from /frida/gum/gumstalker.h:5:
In file included from /frida/gum/arch-x86/gumx86writer.h:5:
/frida/gum/arch-x86/gumx86writer.h:5: error: include file 'gum/gummetalarray.h' not found
/frida/gum/arch-x86/gumx86writer.h:5: error: include file 'gum/gummetalarray.h' not found
    at [anon] (../../../frida-gum/bindings/gumjs/gumdukcmodule.c:128)
    at [anon] (native)
```

**Comment by oleavr** [comment]
Thanks for noticing! This was an embarrassing regression. Fixed in 64a99e6f070e96d84cac6e4a013d4a209c646a6e. Releasing 12.8.1 now with this fix.

**Comment by rozbo** [comment]
It's still like this in 12.8.20

**Comment by oleavr** [comment]
Fixed in 6f04791. Thanks for the heads up, @rozbo!
